src/Microsoft.Health.Core/Extensions/StringExtensions.cs

```diff
@@ -22,6 +22,6 @@ public static string ComputeHash(this string data)
         EnsureArg.IsNotNull(data, nameof(data));
 
         byte[] hash = SHA256.HashData(Encoding.UTF8.GetBytes(data));
-        return BitConverter.ToString(hash).Replace("-", string.Empty, StringComparison.Ordinal);
```

Comment:
`BitConverter.ToString(string)` returns values like `AB-CD-EF-01` but `Convert.ToHexString` skips the `-` entirely